src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntegerTests.cs

```diff
@@ -201,6 +201,7 @@ private static void VerifyTypes(INamedTypeSymbol underlyingType, INamedTypeSymbo
             Assert.Empty(nativeIntegerType.Interfaces);
 
             Assert.NotSame(underlyingType, nativeIntegerType);
+            Assert.Same(underlyingType, nativeIntegerType.NativeIntegerUnderlyingType);
```

Comment:
>NativeIntegerUnderlyingType [](start = 58, length = 27)

I think we should also define and test behavior of this API with relation to NullableAnnotations

Reply:
> I think we should also define and test behavior of this API with relation to NullableAnnotations

It doesn't look like this is addressed. No tests for NullableAnnotation/WithNullableAnnotation

---
In reply to: [385435337](https://github.com/dotnet/roslyn/pull/41970#discussion_r385435337) [](ancestors = 385435337)